keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -504,20 +537,174 @@ Version skew is possible among the following components in this case: K8s API se
 
 Once this feature is implemented in the API server there is a chance that the CPI implementation has to deal with load balancer Services with mixed protocol configuration anytime, even if the API server is downgraded later. The CPI implementation shall be prepared for this, i.e. the CPI implementation cannot expect anymore that the API Server (or any other component, like the Service Controller) filters out such Service definitions. If the CPI implementation wants to have such filtering it has to implement that on its own. In this case the CPI implementation shall be upgraded before the feature is enabled on the API server. In case of a rollback of the API Server such Services can still exist in the cluster, so the CPI implementation should not be downgraded to a version that does not implement that filtering. This is the reason why the CPI implementations shall be updated (if necessary) to be able to deal with such Service definitions already in the first release of this feature.
 
-## Implementation History
+## Production Readiness Review Questionnaire
```

Comment:
It looks like this section is all boilerplate?

Reply:
The PRRQ is required and I added my comments there where applicable in alpha phase.